LayoutTests/svg/crash-svg-filter-empty-viewport.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="0" height="0" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" onload="test()">
```

Comment:
No need for `xmlns:xlink`. This namespace was deprecated in SVG2.

Reply:
Does the SVG have to have zero size `width="0" height="0"` to reproduce the bug?

Reply:
> No need for `xmlns:xlink`. This namespace was deprecated in SVG2.

Removed.